### PR TITLE
core/ts-version_issue_449_remove_supported_networks

### PR DESCRIPTION
**Pull request by GreatGaryCooper**
## Check list  

- [x] All Merge Request pipelines should be passed.
- [x] Make sure all the changes are necessary and correct.
- [x] Make sure no needed change missing (all necessary changes are made).
- [x] Merge Request should contains only changes for one purpose (one feature or bug or refactoring).
- [x] All necessary unit tests are written.
- [x] Critical changes should be tested locally (or on staging).

### CODE REVIEW

File: ts-version/src/data/database/postgres/migration/002-init-fills-table.migration.ts [hunk 1]
```diff
@@ -1,10 +1,8 @@
 import { QueryInterface, DataTypes, literal } from "sequelize";
-import { FillSide } from "../../../../domain/types/fill.type";
+import { FillSides } from "../../../../domain/types/fill.type";
```
Comment (WilliamHawkingStephen) [inline ts-version/src/data/database/postgres/migration/002-init-fills-table.migration.ts:2]: maybe we can use path mapping for this import:
`import { FillSides } from "@type/fill.type";`

File: ts-version/src/presentation/http/controllers/fill.controller.ts [hunk 2]
```diff
@@ -38,18 +38,18 @@ export class FillController extends BaseController {
...
-      market: market as string,
-      fillId: Number(fill_id),
-      side: side as FillSide,
-      startTime: start_time as unknown as Date,
-      endTime: end_time as unknown as Date,
-      limit: limit as unknown as number,
+      market: market,
+      fillId: fill_id != undefined ? Number(fill_id) : undefined,
+      side: side,
+      startTime: start_time != undefined ? new Date(start_time) : undefined,
```
Comment (WilliamHawkingStephen) [inline ts-version/src/presentation/http/controllers/fill.controller.ts:50]: we can add this change to order controller (getByUser) and i think maybe we should set which parameter should pas as startTime and endTime 
- like millisecond , etc

File: ts-version/src/data/database/postgres/migration/002-init-fills-table.migration.ts (hunk 1 above)
Comment (hypersolo) [reply ts-version/src/data/database/postgres/migration/002-init-fills-table.migration.ts:2]: migrations won't work by importing files with paths.

**Comment by hypersolo** [comment]
as @WilliamHawkingStephen said, order controller changes are missed.
and also, fillSide change must be applied to other enum types too.

**Review by hypersolo — APPROVED** [review]

**Pull request merged**